.github/workflows/claude.yml

```
@@ -36,15 +41,90 @@ jobs:
+            mcp__tavily__*,
+            mcp__supabase__*,
+            mcp__shadcn__*,
+            Read,
+            Write,
+            Edit,
+            Glob,
+            Grep,
+            Bash,
+            Task,
+            ExitPlanMode,
+            TodoWrite
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)